package.json

```diff
@@ -90,7 +90,7 @@
     "react": "16.8.6",
     "react-collapsible": "2.3.2",
     "react-dom": "16.8.6",
-    "react-i18next": "8.1.0",
+    "react-i18next": "^11.5.0",
```

Comment:
let's use exact version for this one

Reply:
Oh, right, thanks - didn't notice that, will fix.